src/js/views/patients/sidebar/action/action-sidebar.hbs

```diff
@@ -16,9 +16,9 @@
     <div class="flex u-margin--t-16"><h4 class="action-sidebar__label u-margin--t-8">{{ @intl.patients.sidebar.action.actionSidebarTemplate.stateLabel }}</h4><div class="flex-grow" data-state-region></div></div>
     <div class="flex u-margin--t-8"><h4 class="action-sidebar__label u-margin--t-8">{{ @intl.patients.sidebar.action.actionSidebarTemplate.ownerLabel }}</h4><div class="flex-grow" data-owner-region></div></div>
     <div class="flex u-margin--t-8"><h4 class="program-action-sidebar__label u-margin--t-8">{{ @intl.admin.sidebar.action.actionSidebarTemplate.dueDayLabel }}</h4>
-      <div class="flex flex-grow program-action-sidebar__due">
-        <div class="flex-grow" data-due-day-region></div>
-        <div class="flex-grow" data-due-time-region></div>
+      <div class="flex flex-grow program-action-sidebar__due">{{~ remove_whitespace ~}}
```

Comment:
The `.program-action-sidebar_due` and `program-action-sidebar__label` classes should only be used in the program section.  This file should be using `action-sidebar__` classes